src/components/icons/arrow-index-icon/arrow-index-icon.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+export const ArrowIndexIcon = ({ className }) => (
```

Comment:
I think this should call ArrowIcon, the name of the component should be independent of where it is used